packages/ui/src/components/Switch/switch.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    Switch as DefaultSwitch,
+    SwitchProps as DefaultSwitchProps,
+} from 'antd'
+import React, { useCallback } from 'react'
+
+import { sendAnalyticsChangeEvent } from '../_utils/analytics'
+
+import './style.less'
+
+
+const SWITCH_CLASS_PREFIX = 'condo-switch'
+
+export type SwitchProps = Pick<DefaultSwitchProps,
+'className' |
+'id' |
+'autoFocus' |
+'checked' |
+'defaultChecked' |
+'disabled' |
+'size' |
```

Comment:
I guess we should map our design system sizes (small / ~~medium~~ / large) to antd (small / default), shouldn't we?